README.md

```diff
@@ -23,33 +23,54 @@
     $ npm run docker:db
     ```
 
-5. Create all the necessary tables in database based on Prisma schema, also this command seed data into database.
+5. Create all the necessary tables in database based on Prisma schema.
     ```bash
     $ npx prisma migrate dev
     ```
 
 # Start the app
 
-```bash
-# development
-$ npm run start
+1.  Run the container with db your build earlier]
+
+    ```bash
+    $ npm run docker:db
+    ```
 
-# watch mode
-$ npm run start:dev
+2.  Create all the necessary tables in database based on Prisma schema.
 
-# production mode
-$ npm run start:prod
-```
+    ```bash
+    $ npx prisma migrate dev
+    ```
+
+3.  Seed the database with data
+    !!! If you are going to start your app in production mode, skip this step !!!
```

Comment:
``npx prisma migrate dev`` - dev for development mode so, in this case we don't have production mode and don't need do seed, because command `npx prisma migrate dev` also do  seed